src/core/geometry/fuzzyfactory/FuzzyFactory3.test.js

```diff
@@ -0,0 +1,36 @@
+const test = require('ava')
+
+const { EPS } = require('../../constants')
+const poly3 = require('../poly3')
+
+const FuzzyFactory3 = require('./FuzzyFactory3')
+
+const { comparePolygons } = require('../../../../test/helpers/index')
+
+test('FuzzyFactory3: factories for 3D geometries should function properly', (t) => {
```

Comment:
@z3dev can we please clarify what 'function properly' means ? we should prefer some explicit things like `FuzzyFactory3: factories for 3D geometries should return a single polygon when merging 2 polygons`  etc